qiime2/sdk/result.py

```diff
@@ -185,7 +217,7 @@ def validate(self, level=NotImplemented):
 
 
 class Artifact(Result):
-    extension = '.qza'
+    extension = 'qza'
```

Comment:
I think this should be rolled back - the business logic implemented in `Result.save()` handles this case just fine.

```suggestion
    extension = '.qza'
```